src/config.rs

```diff
@@ -1391,7 +1391,7 @@ fn rgb_from_hex<'a, D>(deserializer: D) -> ::std::result::Result<Rgb, D::Error>
     match rgb {
         Ok(rgb) => Ok(rgb),
         Err(err) => {
-            error!("problem with config: {}; Using color #ff00ff", err);
+            error!("Problem with config: {}; Using color #ff00ff", err);
```

Comment:
This might be a bit pedantic, but since being pedantic is kinda the point of this PR, I think it might be worth thinking about a standardized way to specify values. The `#ff00ff` is not quoted in any kind here for example. In an earlier log message, a `true` value is surrounded by grave quotes, the `indexed_color` log message quotes the specified index, but doesn't quote the index it expects you to be above.

Reply:
How do you feel about back ticks. I personally find them in style.

Reply:
@chrisduerr so what are we considering a "value" then. I'd be tempted to wrap a lot of things being formatted with either `{}` or `{:?}` as values. Do you have an idea how we'd do this somewhat systematically?

Reply:
Ok, so almost every value inside `{}` or `{:?}` has been wrapped, with a few exceptions, mainly when the value is delimited in another way.